api/models.py

```diff
@@ -170,6 +170,9 @@ class Activity(Base):
     __tablename__ = 'activities'
     activity_type_id = db.Column(db.String,
                                  db.ForeignKey('activity_types.uuid'))
+    activity_date = db.Column(db.Date)
+    added_by = db.Column(db.String,
```

Comment:
This should be `added_by_id` so that you can add in an `added_by` relationship.

Reply:
@LarryWachira Done. Changed it to `added_by_id `